examples/auth/main.go

```diff
@@ -4,30 +4,44 @@
 package main
 
 import (
+	"bytes"
 	"fmt"
+	"net/http"
 	"os"
 
-	"storj.io/storj/netstate/auth"
-
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
+	//"storj.io/storj/netstate/auth"
 )
 
 // example of how the auth package is working.
 // see readme in auth/ for how to run
 func main() {
 
+	// set client api credentials
+	// hardcoded as an example
```

Comment:
technically the below is the opposite of hardcoded. hardcoded would be more like

```
req.Header.Add("X-Api-Key", "abcd43531")
```

Reply:
but whatever!